commands/health

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -eu
+
+#
+# CLI: System health information
+#
+
+# @sudo cmd_inventory admin,operator
+# @doc cmd_inventory
+# System inventory
+function cmd_inventory {
+  lsinventory "$@"
+}
+
+# @sudo cmd_sensors admin,operator
+# @doc cmd_sensors
+# Sensor readings
+function cmd_sensors {
+  lssensors "$@"
+}
+
+# @doc cmd_logs
+# Operations with logs
+function cmd_logs {
+  subcommand "$@"
+}
+
+# @doc cmd_logs_list
+# Get list of system logs
+#function cmd_logs_list {
+#  not implemented
+#}
+
+# @sudo cmd_logs_show admin,operator
+# @doc cmd_logs_show
+# Get system logs
+function cmd_logs_show {
+  expect_noarg "$@"
```

Comment:
`-f|--follow` is very useful.

Reply:
Done according to the internal document.

Reply:
Do we really need to implement the third way to show uptime? :)
Vesnin calls dreport plugins which behavior depends on dreport library and we need to implement this library too. The second problem - dreport scripts doesn't intend to work under sudo and we have to audit every change in this project to be sure that there are no any security vulnerabilities.

Here the list of plugins:
E10diskusage       E1pnorversion      E30ipaddr          E5chassisstate
E10failedservices  E20audit           E30iplink          E5hostnamectl
E10journalpretty   E20elogall         E40fwprintenv      E5hoststate
E10meminfo         E20hostlogger      E50cpuinfo
E10top             E20inventory       E50uptime
E1osrelease        E25obmcconsole     E5bmcstate

Most of them are already implemented by other commands. And I don't understand how CPU info or Top relates to the `log` command.